sympy/physics/units/definitions.py

```diff
@@ -323,17 +323,18 @@
 # Newton constant
 G = gravitational_constant = Quantity("gravitational_constant", abbrev="G")
 gravitational_constant.set_dimension(length**3*mass**-1*time**-2)
-gravitational_constant.set_scale_factor(6.67408e-11*m**3/(kg*s**2))
+gravitational_constant.set_scale_factor(6.67430e-11*m**3/(kg*s**2))
```

Comment:
This constant also has some extra digits in parentheses. I don't know if those should be included.

Reply:
My personal preference would be to not include them since they are usually the most likely to change.

Reply:
Look at this:

https://physics.nist.gov/cgi-bin/cuu/Value?k|search_for=abbr_in!

The additonal digits is the uncertainty and it should be be concatenated, instead, here, the value is between ...30-15 and ...30+15. I did the same interpretation earlier.

Reply:
Non-clickable link, but https://physics.nist.gov/cuu/Constants/index.html and click your way through...

Reply:
Oh, I see... They aren't additional digits, that's the one sigma window for the last two digits. Thanks for pointing that out! They definitely shouldn't be included then.